spec/features/merchants/dashboard_spec.rb

```diff
@@ -144,4 +144,32 @@
       expect(page).to have_content(@merchant_1.top_5_customers[4].transaction_count)
     end
   end
-end
+
+  describe 'in items ready to ship section' do 
+    scenario 'i see a list of item names that have not been shipped' do 
+      expect(page).to have_content("Items Ready to Ship")
+
+      expect(page).to have_content(@item_1.name)
+      expect(page).to have_content(@item_2.name)
+      expect(page).to have_content(@item_3.name)
+      expect(page).to have_content(@item_4.name)
+      expect(page).to have_content(@item_5.name)
+    end
+
+    scenario 'i see the invoice id next to the item name' do 
+save_and_open_page
+      expect(page).to have_link(@invoice_item_1.invoice_id, href: merchant_invoice_path(@merchant_1.id, @invoice_1.id))
+      expect(page).to have_link(@invoice_item_2.invoice_id, href: merchant_invoice_path(@merchant_1.id, @invoice_2.id))
+      expect(page).to have_link(@invoice_item_3.invoice_id, href: merchant_invoice_path(@merchant_1.id, @invoice_3.id))
+      expect(page).to have_link(@invoice_item_4.invoice_id, href: merchant_invoice_path(@merchant_1.id, @invoice_4.id))
+      expect(page).to have_link(@invoice_item_5.invoice_id, href: merchant_invoice_path(@merchant_1.id, @invoice_5.id))
+    end 
+  end
```

Comment:
Looking at this test and looking at the user story I'd say the test doesn't exactly show that the piece of information asked for is next to the item name. There are ways we could make this test more robust by using within blocks or checking the order of text on the page . For example, we could write an orderly test that checks that an item name appears before the associated index number, and that that index number appears before the next expected item name, and that that item name appears before that item's associated index number. 

I don't think it is IMPERATIVE that we do that here, and we are always able to verify the information visually with a save_and_open_page, but as we continue to write tests it might be good for us to think about expanding what we are testing, not just testing that a page has some text on it, but that it is grouped the way we think it is.